src/guides/v2.3/frontend-dev-guide/layouts/layout-create.md

```diff
@@ -17,14 +17,24 @@ If the new page has a `3-columns-double-footer` layout, create a custom page-lay
     <update handle="3columns"/>
     <referenceContainer name="page.wrapper">
         <container name="footer-bottom" as="footer-bottom" after="footer" label="Footer Bottom" htmlTag="footer" htmlClass="page-footer-bottom">
-            <container name="footer-bottom-content" as="footer-bottom-content" htmlTag="div" htmlClass="footer content">
-                <block class="Magento\Framework\View\Element\Template" name="report.bugs.bottom" template="Magento_Theme::html/bugreport.phtml"/>
-            </container>
+            <container name="footer-bottom-content" as="footer-bottom-content" htmlTag="div" htmlClass="footer content" />
```

Comment:
Does this container tag need a closing tag? There is only one `</container>`

Reply:
The "container" can have closing tag <container ... > </container> or can be salf closing <container ... />
The examples: _view/adminhtml/page_layout/admin-1column.xml_ line: 11
`<container name="after.body.start" as="after.body.start" label="Page Top" before="-"/>`